### ISSUE TITLE
xdotool is too old on DEBIAN based distro

### ISSUE THREAD

**Issue by EliteEgle** [jordansissel/xdotool#395]
Hello jordansisel,
I was using [Waishnav/Watcher](https://github.com/waishnav/watcher) to analyse my screen-time
But just because debian xdotool  version is old ```xdotool getactivewindow getwindowclassname``` command is not working.
and hence not getting active window to program
its silly to ask but I've to ask "Can you update debian package?" if you have that permission  

Note: On my arch machine it works just fine

**Comment by jordansissel** [comment]
I don't use Debian myself, but it looks like the latest version [available on any Debian version](https://tracker.debian.org/pkg/xdotool) is 3.20160805.1, and `getwindowclassname` was added in version 3.20210804.1. (For what it's worth, there was a 5 year gap between xdotool releases, so Debian is only a few months behind)

@dkg Thoughts? No pressure, though! Is this waiting on #387?

**Comment by EliteEgle** [comment]
No, but the following command is not working ```xdotool getactivewindow getwindowclassname``` on ubuntu whereas on arch and void linux its working

**Comment by dkg** [comment]
Yeah, i tried to update xdotool and libxdo in debian and ran into the ABI breakage noted in #387, so i stopped because i wasn't sure what the right next steps were given the changes i discovered.  I didn't want to unilaterally bump the SONAME without it being bumped upstream, and i wasn't sure whether it made sense to patch it further to keep the ABI stable without any consensus that this was the right thing to do.

If there were a clearer plan in place, i'd be game to try again.
